docs/release-notes/8.15.asciidoc

```diff
@@ -1,6 +1,43 @@
 [[release-notes-header-8.15.0]]
 == 8.15
 
+[discrete]
+[[release-notes-8.15.4]]
+=== 8.15.4
+
+[discrete]
+[[known-issue-8.15.4]]
+==== Known issues
+
+// tag::known-issue-189676[]
+[discrete]
+.Tags appear in Elastic AI Assistant's responses
+[%collapsible]
+====
+*Details* +
+On August 1, 2024, it was discovered that Elastic AI Assistant's responses when using Bedrock Sonnet 3.5 may include `<antThinking>` tags, for example `<search_quality_reflection>` ({kibana-issue}189676[#189676]).
+
+====
+// end::known-issue-189676[]
+
+[discrete]
+[[enhancements-8.15.4]]
+==== Enhancements
+* Enhances {elastic-defend} by improving the `call_stack_final_user_module` attribution where potential `proxy_call` modules are encountered during Windows call stack analysis.
+
+[discrete]
+[[bug-fixes-8.15.4]]
+==== Bug fixes
+* Fixes a conflict that can result in a Windows boot failure `0xC000007B` for `ElasticElam.sys` when {elastic-defend} 8.15.2 or 8.15.3 is installed alongside CrowdStrike.
+* Fixes a bug that caused an Elastic AI Assistant error if you had over 20 conversations and tried to access or update any of them ({kibana-pull}197305[#197305]).
+* Makes Automatic Import more forgiving if LLMs return ECS mappings in unexpected formats ({kibana-pull}195167[#195167]).
+* Fixes a bug that caused fields from all indices to display when adding a filter to a rule that you were editing. Now, only fields from the rule's specified indices appear ({kibana-pull}194678[#194678]).
```

Comment:
same PR fixed also this issue: https://github.com/elastic/kibana/issues/181643
Would it be worth to mention it?

Reply:
Sure! I can reference 181643 in this summary as well.

```suggestion
* Fixes a bug that caused fields from all indices to display when adding a filter to a rule that you were editing. Now, only fields from the rule's specified indices appear ({kibana-pull}194678[#194678], {kibana-pull}181643[#181643]).
```

Reply:
@nastasha-solomon , there is no doc instance to check change in this PR, but https://github.com/elastic/kibana/issues/181643 is issue. Does it work well `{kibana-pull}181643[#181643]` as pull request?

Reply:
Hey @vitaliidm, just saw this now. We don't typically link to issues in the fixed bug section of the release notes, but it's possible to do so if needed. I'll open a new PR to fix this reference.

Reply:
This should be fixed in https://github.com/elastic/security-docs/pull/6126. Thanks for reporting this @vitaliidm!